base_attachment_object_storage/models/ir_attachment.py

```diff
@@ -18,6 +19,10 @@
 _logger = logging.getLogger(__name__)
 
 
+def is_true(strval):
+    return bool(strtobool(strval or '0'))
```

Comment:
Can we use `str2bool` provided by Odoo instead?
https://github.com/odoo/odoo/blob/14.0/odoo/tools/convert.py#L206-L207=